### PR TITLE
Add apex test -- Install/Update/Uninstall vulnerable package from PMUI

### PR DESCRIPTION
**Pull request by CiciLi1**
## Bug

Fixes: https://github.com/NuGet/Client.Engineering/issues/2388

Regression? Last working version:

## Description

Add apex test -- Install/Update/Uninstall vulnerable package from PMUI to verify that the installed/updated/uninstalled package "Newtonsoft.Json" is vulnerable.
## PR Checklist

- [X] PR has a meaningful title
- [X] PR has a linked issue.
- [X] Described changes

- **Tests**
  - [X] Automated tests added
  - **OR**
  
  - [ ] Test exception
  - **OR**
  - [X] N/A 

- **Documentation**
  
  - [ ] Documentation PR or issue filled
  - **OR**
  - [X] N/A

### CODE REVIEW

File: test/NuGet.Tests.Apex/NuGet.PackageManagement.UI.TestContract/ApexTestUIProject.cs [hunk 1]
```diff
@@ -69,6 +69,12 @@ public bool VerifyFirstPackageOnTab(string tabName, string packageId, string pac
             }
         }
 
+        public bool VerifyVulnerablePackageOnTab()
```
Comment (nkolev92) [inline test/NuGet.Tests.Apex/NuGet.PackageManagement.UI.TestContract/ApexTestUIProject.cs]: naming: similar feedback as the previous PR.  It's verifying the top package so the name sshould communicate that.

Comment (CiciLi1) [reply test/NuGet.Tests.Apex/NuGet.PackageManagement.UI.TestContract/ApexTestUIProject.cs]: Thanks for the suggestion, I've updated the name. Please help to review it again. Thanks.

File: test/NuGet.Tests.Apex/NuGet.Tests.Apex/Apex/NuGetUIProjectTestExtension.cs [hunk 2]
```diff
@@ -38,6 +38,18 @@ public void AssertSearchedPackageItem(string tabName, string packageId, string p
             searchPackageResult.Should().BeTrue($"searching for the package {packageId} in the {tabName} tab failed");
         }
 
+        public void AssertVulnerablePackage()
```
Comment (nkolev92) [inline test/NuGet.Tests.Apex/NuGet.Tests.Apex/Apex/NuGetUIProjectTestExtension.cs]: same naming feedback. It should apply everywhere.

**Review by nkolev92 — DISMISSED** [review]
One small naming suggestion and then good to go.

**Comment by CiciLi1** [comment]
> One small naming suggestion and then good to go.

Hi @jebriede, I have updated the name from `AssertVulnerablePackage` to `AssertInstalledPackageVulnerable` according to Nikolche's suggestion, could you please help to review and approve this PR, thanks!

**Review by v-luzh — APPROVED** [review]

**Review by jebriede — APPROVED** [review]

**Pull request merged**